eng/pipelines/templates/stages/platform-matrix-live.json

```diff
@@ -5,7 +5,7 @@
   "include": [
     {
       "StaticConfigs": {
-        "macOS-11": {
+        "macos-latest": {
```

Comment:
nit: The pipeline name on GitHub CI shows as "macoslatest" which is hard to parse, with it all being lower same case.

Can we name it something like `macOSlatest` or even `macOS-latest`?

Reply:
@benbp I suspect we could do this with the displayNames property of the configuration correct?

Reply:
@benbp how can we update the pipeline name to be `MacOS-latest` instead of `macoslatest`?